tests/unit/bots/test_bots.py

```diff
@@ -0,0 +1,628 @@
+import datetime
```

Comment:
Test logic itself is the same, changed some things to make it work:

- changed all from sqlalchemy models to django models
- got rid of `dbsession`
- adjusted related manager calls to be .all() (and wrapped them in `list` when asserted against a list). This was for repo.github_app_installations and repo.author
- added django_db decorator
- 
-